scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
Was just watching that video again and noticed him meleeing the adds, so I looked at how much hp they lost and how much damage he did. The one I saw was from 46% -> 31% with a hit of 522. So 15% of their health was 522. 522 * 100 / 15 = maxHP. maxHP = 3480 (probably 3500). They're currently set at 9k. Thoughts?